### PR TITLE
Add Shawnee county Kansas 2016 shapefile

### PR DESCRIPTION
**Pull request by karpet**
Finally hacked together a EPSG:4326 shapefile from the ArcGIS output at http://gis.snco.us/arcgis2/rest/services/Elections/ElectionsNewAllDistricts/MapServer

### CODE REVIEW

File: scripts/20-kansas.sh [hunk 1]
```diff
@@ -32,6 +32,13 @@ ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Sedgwick' as county, PR
         -t_srs EPSG:4326 -f GPKG -nln state -append \
         $TEMPDIR/state.gpkg data/20-kansas/2016/20173-sedgwick/precincts.geojson
 
+#
+# Add Shawnee County (FIPS 227) to the statewide Geopackage file.
+#
+ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Shawnee' as county, PRECINCT AS precinct, 'polygon' AS accuracy FROM OGRGeoJSON" \
```
Comment (nvkelso) [inline scripts/20-kansas.sh]: I think `'Shawnee'` here should be `227` instead.

**Comment by nvkelso** [comment]
Looks good other than the one FIPS code comment. Thank you!

File: scripts/20-kansas.sh (hunk 1 above)
Comment (karpet) [reply scripts/20-kansas.sh]: Is that true just for that particular line? All the other county-specific commands above it use the county name, not the FIPS code, for the sql statement.

**Comment by karpet** [comment]
@nvkelso - think I responded to your question in 7251b2b

File: scripts/20-kansas.sh [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 # Create a new statewide Geopackage file in $TEMPDIR/state.gpkg.
 # Gunzip to a temporary GeoJSON file along the way.
-# Skip Douglas County (FIPS 045) since it'll come from another file.
+# Skip some counties that have newer files.
 #
 CURDIR=`dirname $0`
 TEMPDIR=`$CURDIR/tmpdir.py kansas-`
 
-ogr2ogr -sql "SELECT '2012' AS year, 'Kansas' AS state, SUBSTR(VTD_2012, 3, 3) AS county, VTD_2012 AS precinct, 'polygon' AS accuracy FROM OGRGeoJSON WHERE SUBSTR(VTD_2012, 3, 3) NOT IN ('045', '091', '173', '209')" \
+ogr2ogr -sql "SELECT '2012' AS year, 'Kansas' AS state, SUBSTR(VTD_2012, 3, 3) AS county, VTD_2012 AS precinct, 'polygon' AS accuracy FROM OGRGeoJSON WHERE SUBSTR(VTD_2012, 3, 3) NOT IN ('045', '091', '173', '209', '227')" \
```
Comment (nvkelso) [inline scripts/20-kansas.sh]: `20` instead of `Kansas` for state, please

File: scripts/20-kansas.sh [hunk 3]
```diff
@@ -32,6 +32,13 @@ ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Sedgwick' as county, PR
         -t_srs EPSG:4326 -f GPKG -nln state -append \
         $TEMPDIR/state.gpkg data/20-kansas/2016/20173-sedgwick/precincts.geojson
 
+#
+# Add Shawnee County (FIPS 227) to the statewide Geopackage file.
+#
+ogr2ogr -sql "SELECT '2016' AS year, 'Kansas' AS state, 'Shawnee' as county, NAME AS precinct, 'polygon' AS accuracy FROM OGRGeoJSON" \
```
Comment (nvkelso) [inline scripts/20-kansas.sh]: `20` instead of `Kansas` for state, please

Comment (nvkelso) [reply scripts/20-kansas.sh]: `20227` instead of `Shawnee` for county, please

Comment (nvkelso) [reply scripts/20-kansas.sh]: And for "precinct" it should probably be concat of `20227` and `PrecinctIdentifier` (which is the ID versus the name that has the identifier postfix).

Comment (karpet) [reply scripts/20-kansas.sh]: To be clear, the rest of the counties use `Kansas` for state and the county name, not FIPS, for county. I should change all of them, not just Shawnee?

Comment (karpet) [reply scripts/20-kansas.sh]: For precinct, there seems to be a mix in the other KS counties, but a preference toward using the VTD code. That's part of `PRECINCTID` in Shawnee, so I'll use that. Unless you want the name and VTD concat?

**Pull request merged**